### ISSUE TITLE
The value after --intercept-gencov flag is the correct intercept term

### ISSUE THREAD

**Issue by chenyan53535** [bulik/ldsc#87]
Hi, I want to estimate the genetic correlation using the correct intercept term because I have the individual phenotype info and N1, N2, Ns.  Genetic correlation paper says that I need to use --constrain-intercept flag to set the correct intercept term. But in this version I did not find this parameter. I think --intercept-gencov flag may be used for setting correct intercept term, could you please tell me if what I think is correct? Thank you.

**Comment by rkwalters** [comment]
Correct, --intercept-gencov is this the flag you are looking for. It was probably renamed at some point to avoid confusion with constraining the univariate intercept.

I should note that even with known sample overlap we’ll generally recommend leaving the intercept unconstrained. LD score regression remains a very simple model, and constraining the intercept amounts to a very strong assumption on top of that simplified model. At the very least it’s worth fitting the unconstrained model to verify that the fitted estimate for the intercept is consistent with your expectations.

Cheers,
Raymond

**Comment by chenyan53535** [comment]
Thank you for your help
